packages/vanilla-extract/stories/Text.css.ts

```diff
@@ -58,6 +62,25 @@ export const responsiveText = createTextStyle(
   'responsiveText',
 );
 
+const containerName = createContainer();
+
+export const container = style({
+  containerName,
+  containerType: 'inline-size',
+  outline: 'solid 1px blue',
+});
+
+export const containerText = createTextStyle(
+  textValues.mobile,
+  {
+    '@container': {
+      [`${containerName} (min-width: 400px)`]: textValues.tablet,
+      [`${containerName} (min-width: 600px)`]: textValues.desktop,
+    },
+  },
+  'responsiveText',
```

Comment:
Let's align the `debugId` here:
```suggestion
  'containerText',
```